test/unit/support.js

```diff
@@ -284,7 +295,28 @@ testIframe(
 				"radioValue": true,
 				"reliableMarginLeft": true,
 				"reliableTrDimensions": true,
-				"scrollboxSize": true
+				"scrollboxSize": true,
+				"trim": true
+			},
+			android_4_0: {
+				"ajax": true,
+				"boxSizingReliable": true,
+				"checkClone": false,
+				"checkOn": false,
+				"clearCloneStyle": true,
+				"cors": true,
+				"createHTMLDocument": true,
+				"focusin": false,
+				"noCloneChecked": true,
+				"option": true,
+				"optSelected": true,
+				"pixelBoxStyles": false,
+				"pixelPosition": false,
+				"radioValue": true,
+				"reliableMarginLeft": false,
+				"reliableTrDimensions": true,
+				"scrollboxSize": true,
+				"trim": false
```

Comment:
All the other browser entries are put from newest to oldest within the same family so let's put this after the `android` entry for consistency.